CrowdFundingApplication/wwwroot/js/ModalWindow.js

```diff
@@ -0,0 +1,33 @@
+﻿let modal;
+document.addEventListener("click", (e) => {
+    if (e.target.className === "modal-open") {
+        modal = document.getElementById(e.target.dataset.id);
+        openModal(modal);
+    } else if (e.target.className === "modal-close") {
+        closeModal(modal);
+    } else {
+        return;
+    }
+});
+
+const openModal = (modal) => {
```

Comment:
https://getbootstrap.com/docs/4.0/components/modal/

Reply:
In my case component bootstrap modal window did not work that's why I was using my custom modal window